lib/survey_gizmo/rest_response.rb

```diff
@@ -18,6 +18,9 @@ def initialize(rest_response)
 
     # Handle really crappy [] notation in SG API, so far just in SurveyResponse
     (data.is_a?(Array) ? data : [data]).each do |datum|
+      # Fix for when SG API returns [null] as data
+      next if dataum.is_a?(NilClass)
+      
```

Comment:
A much cleaner approach would be to use `Array(data).each do |datum|`
It replaces the previous code, it converts single value into one element array and ignores nil value:
```ruby
>> Array(nil)
=> []
>> Array(42)
=> [42]
>> Array([1, 2, 3])
=> [1, 2, 3]
```